tests/test_pipelines/test_rasterize.py

```diff
@@ -71,49 +63,51 @@ def add_vector_data(pipeline):
         eopatch.save(eopatch_folder, features=[(FeatureType.VECTOR_TIMELESS, "LULC_VECTOR")], overwrite_permission=1)
 
 
+def reset_output_folder(config_path: str) -> Optional[str]:
+    raw_config = interpret_config_from_dict(collect_configs_from_path(config_path)[0])
+    pipeline = load_pipeline_class(raw_config).from_raw_config(raw_config)
+
+    output_folder_key: Optional[str] = raw_config.get("output_folder_key")
+    if output_folder_key is not None:
```

Comment:
the `raw_config.get` returns `Any | None`, so I had to write it like this to avoid mypy issues, but otherwise correct, it can never be none in this test.

Should I ignore the errors?

Reply:
why not `raw_config["output_folder_key"]`?

Reply:
cause I'm a dum-dum!

Reply:
fixed